packages/react-native-editor/ios/GutenbergDemo/GutenbergViewController.swift

```diff
@@ -396,6 +397,15 @@ extension GutenbergViewController {
                 self.toggleHTMLMode(action)
         })
     }
+
+    var removeAllBlocksAction: UIAlertAction {
+        return UIAlertAction(
+            title: "Remove All Blocks",
+            style: .default,
+            handler: { [unowned self] action in
```

Comment:
```suggestion
            handler: { [unowned self] _ in
```

The `action` closure parameter is unused so can be omitted.